### PR TITLE
Release types

### PR DESCRIPTION
**Pull request by gjolly**

### CODE REVIEW

File: all-clouds/all-clouds-explanation/release-types.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Canonical does not publish daily images on all public clouds and for all products. If a cloud-image or offer is not explicitely tagged as "daily", one can safely assume that it is a "release" image/offer.
+
+New Release images are published when a package affecting the VM first boot experience of the image is updated in the archive. THe release package set includes;
+
+ * the linux kernel
+ * systemd
+ * glibc
+ * grub
+ * openssl
+ * cloud-init
```
Comment (j5awry) [inline all-clouds/all-clouds-explanation/release-types.rst:15]: * cloud agents (different per cloud)

**Review by j5awry — COMMENTED** [review]
one addition. I'll leave nits to someone with a better copy-editing eye.

File: all-clouds/all-clouds-explanation/release-types.rst (hunk 1 above)
Comment (gjolly) [reply all-clouds/all-clouds-explanation/release-types.rst:13]: done

File: all-clouds/all-clouds-explanation/release-types.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Cloud-image release types
+=========================
+
+Canonical publishes two different class of cloud-images: daily images and release images. While release images are fully tested and production grade, daily images are preview images containing all the latest updates from the archive. Daily images are test images and are not suitable for use in production.
+
+Canonical does not publish daily images on all public clouds and for all products. If a cloud-image or offer is not explicitely tagged as "daily", one can safely assume that it is a "release" image/offer.
+
+New Release images are published when a package affecting the VM first boot experience of the image is updated in the archive. THe release package set includes;
```
Comment (toabctl) [inline all-clouds/all-clouds-explanation/release-types.rst:8]: sTHe/The/

Comment (toabctl) [reply all-clouds/all-clouds-explanation/release-types.rst:8]: s/;/:/

**Review by j5awry — APPROVED** [review]

**Pull request closed without merging**